frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.styled.tsx

```diff
@@ -2,11 +2,19 @@ import styled from "@emotion/styled";
 
 import { Ellipsified } from "metabase/core/components/Ellipsified";
 import { Icon } from "metabase/core/components/Icon";
-import { color } from "metabase/lib/colors";
 import { space } from "metabase/styled-components/theme";
+import { isEmpty } from "metabase/lib/validate";
+
+// NIGHT-MODE TEXT HACK:
+// Our "fullscreen-night-text" className only supports one shade of text (white).
```

Comment:
Reimplementing night mode in a more convenient way for our styles is definitely out of the scope of this project.

Can we still avoid such hacks and use nested styles?
```
  .fullscreen-night-text & {
    color: ${...};
  }
```